src/Builder/Encoder/OperatorEncoder.php

```diff
@@ -108,10 +112,17 @@ private function encodeAsSingle(OperatorInterface $value): stdClass
         throw new LogicException(sprintf('Class "%s" does not have a single property.', $value::class));
     }
 
+    /**
+     * Wrap the result in an object if the operator has a name.
+     * The operator name is NULL, and the result is returned as is
+     * when wrapObject is false in the YAML configuration of the operator.
```

Comment:
I think this comment no longer applies, as you now assert that `$value::NAME` is a string.